java/src/org/certificatetransparency/ctlog/MerkleAuditProof.java

```diff
@@ -0,0 +1,18 @@
+package org.certificatetransparency.ctlog;
+
+import org.certificatetransparency.ctlog.proto.Ct;
+
+
+public class MerkleAuditProof {
+  public Ct.Version version;
+  public long treeSize;
+  public long leafIndex;
+  public java.util.List<byte[]> pathNode;
```

Comment:
I think I stole this from somewhere which had somehow decided that this was okay, but sure.